src/main/java/com/derongan/minecraft/deeperworld/DeeperWorld.kt

```diff
@@ -34,5 +36,13 @@ class DeeperWorld : JavaPlugin() {
 
         //register command executor
         DeeperCommandExecutor
+
+        Bukkit.getScheduler().schedule(this){
```

Comment:
With skedule I usually prefer calling the extension function on plugin
```suggestion
        schedule {
```